### PR TITLE
feat: privacy policy and terms & conditions

### PR DESCRIPTION
**Pull request by amrit713**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Showcase

Please attach a picture or a video showing what this change does.

### CODE REVIEW

File: apps/web/src/pages/TermsAndConditions/data.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+export const data = [
```
Comment (ivanms1) [inline apps/web/src/pages/TermsAndConditions/data.ts]: please rename this to `TERMS_AND_CONDITIONS` and move it to the const file.

File: apps/web/src/pages/PrivacyPolicy/PrivacyPolicy.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { useTranslation } from 'next-i18next';
+
+function PrivacyPolicy() {
+  const arr = Array.from(Array(8).keys()).map((_, index) => index + 1);
```
Comment (ivanms1) [inline apps/web/src/pages/PrivacyPolicy/PrivacyPolicy.tsx]: move this outside the component

**Review by ivanms1 — CHANGES_REQUESTED** [review]
Looks good, just small changes. Please add some padding in the areas shown in the images.

Screenshot 2023-07-11 at 9 43 42 AM
Screenshot 2023-07-11 at 9 43 22 AM

**Comment by ivanms1** [comment]
@amrit713 Also there is an error when navigating to the privacy policy page
https://github.com/ivanms1/project-shelf/assets/29688963/7f33bffa-09f4-416c-abb7-a1262ed4759d

File: apps/web/src/pages/TermsAndConditions/data.ts (hunk 1 above)
Comment (revengemiroz) [reply apps/web/src/pages/TermsAndConditions/data.ts]: done

File: apps/web/src/pages/PrivacyPolicy/PrivacyPolicy.tsx (hunk 2 above)
Comment (revengemiroz) [reply apps/web/src/pages/PrivacyPolicy/PrivacyPolicy.tsx]: done

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**